client/src/components/AddVideoForm.js

```diff
@@ -0,0 +1,32 @@
+import { useRef } from "react";
+
+const AddVideoForm = ({ handleAddVideo }) => {
+  const titleRef = useRef(null);
+  const urlRef = useRef(null);
```

Comment:
This is an interesting way of doing it. It's completely valid. Another approach would be to setup a piece of state that is constantly updated to reflect what the user has input (so making your input forms "controlled" inputs).